### DISCUSSION TITLE
GitHub logo is asymmetrical

### DISCUSSION THREAD

**Discussion by tacheometry** [community/community#129012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When downloading the GitHub logo from https://github.com/logos, I noticed that the SVG file had an asymmetrical viewbox (98 width, 96 height).

Then I looked into the SVG file and I realized the width of the `<path>` element is causing this - as what is rendered is 97.63 units wide and 96 units high. I overlaid a circle with the diameter of 96 to visualize the issue:
![image](https://github.com/community/community/assets/39647014/8624c035-cc57-4e6e-8db6-b9de73128180)

So, it seems GitHub's logo does not follow a circle either. That's not an issue, but could the width get fixed to 96, or the height set to 98?

**Comment by realffqq** [comment]
💀

**Comment by theiereman** [comment]
I mean, if it's like that it is probably because there's a reason but, to be fair, it could also be a mistake. 
Asymmetry is not necessarily an issue. For example, lots of text fonts have asymmetrical letters to make them look more balanced.
